MathMagician/Numbers/NaturalNumber.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MathMagician.Numbers
+{
+    public class NaturalNumber : Integer
+    {
+        private int First { get; set; }
+        private List<int> Sequence { get;  set;}
+
+        public NaturalNumber()
+        {
+            First = 1;
+            //this.First = 1;
+            List<int> Sequence = new List<int>();
```

Comment:
@devoisaurus Remove the `List<int>` in order to use your private [property](https://github.com/nss-evening-cohort-04/mathemagician/pull/11/files#diff-806efad2d459e2d24bae6507563a1908R12)